api/exec/compose_stack.go

```diff
@@ -173,3 +176,10 @@ func copyDefaultEnvFile(stack *portainer.Stack, w io.Writer) {
 	}
 	// If couldn't copy the .env file, then ignore the error and try to continue
 }
+
+// copyConfigEnvVarsIfExists write the environment variables from stack configuration to target file
+func copyConfigEnvVarsIfExists(w io.Writer, envs []portainer.Pair) {
+	for _, v := range envs {
+		io.WriteString(w, fmt.Sprintf("%s=%s\n", v.Name, v.Value))
```

Comment:
We should handle the error of this operation.

Nitpick: maybe use `fmt.Fprintf()` directly?